default/python/AI/ProductionAI.py

```diff
@@ -1436,6 +1442,20 @@ def update_stockpile_use():
             planets_in_stockpile_enabled_group.update(group)
 
 
+def empire_has_colony_bld_species(building_name):
+    if not building_name.startswith('BLD_COL_'):
+        return False
+    species_name = 'SP' + building_name.split('BLD_COL_')[1]
```

Comment:
Is this correct? On first sight looks like it should be ```"SP_" + ...```, i.e. currently missing an underscore?